.circleci/config.yml

```diff
@@ -564,16 +500,60 @@ jobs:
       - deploy-stored-build:
           study_key: << parameters.study_key >>
 
-  app-run-tests-job:
+  app-ui-unit-test:
```

Comment:
I need different jobs to run unit tests for specified study and unit tests for ddp-sdk and toolkit. I broke up `app-run-tests-job` job to `app-ui-unit-test` and `toolkit-sdk-unit-test` jobs. Job names are self-explanatory. Splited jobs actually finish quicker because two jobs run in parallel.